### PR TITLE
docs: update code snippet in Sequence.md

### PR DESCRIPTION
**Pull request by shimks**
## Checklist

- [ ] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] API Documentation in code was updated
- [ ] Documentation in [/docs/site](../tree/master/docs/site) was updated
- [ ] Affected artifact templates in `packages/cli` were updated
- [ ] Affected example projects in `packages/example-*` were updated

### CODE REVIEW

**Comment by bajtos** [comment]
> Just wondering if it'd be worthwhile to add a note under `Customizing Sequence Actions` that users using `lb4 app` to generate the `Application` can just modify `src/sequence.ts` instead of having to extend / override the `DefaultSequence`?

+1 from me 👍

File: docs/site/Sequence.md [hunk 1]
```diff
@@ -41,20 +41,17 @@ class DefaultSequence {
 In the example above, `route`, `params`, and `result` are all Elements. When building sequences, you use LoopBack Elements to respond to a request:
 
 - [`Route`](http://apidocs.loopback.io/@loopback%2frest/#Route)
-- [`Request`](http://apidocs.strongloop.com/loopback-next/)  - (TBD) missing API docs link
+- [`Request`](http://apidocs.strongloop.com/@loopback%2frest/#ParsedRequest)  - (TBD) missing API docs link
 - [`Response`](http://apidocs.strongloop.com/loopback-next/) - (TBD) missing API docs link
 - [`OperationRetVal`](http://apidocs.loopback.io/@loopback%2frest/#OperationRetval)
 - [`Params`](http://apidocs.strongloop.com/loopback-next/) - (TBD) missing API docs link
```
Comment (virkt25) [inline docs/site/Sequence.md]: Can you change this to point to `apidocs.loopback.io`. 

- Same for `Request` and `Response` above.

File: docs/site/Sequence.md [hunk 2]
```diff
@@ -83,14 +77,21 @@ class MySequence extends DefaultSequence {
 ## Custom Sequences
 
 Most use cases can be accomplished with `DefaultSequence` or by slightly
-customizing it:
+customizing it. When an app is generated by the command `lb4 app`, a sequence
+file extending `DefaultSequence` is already generated and bound for you so that
+easily customize it.
```
Comment (virkt25) [inline docs/site/Sequence.md]: Can you mention where a user can find this extended version of the DefaultSequence (The file name).

**Comment by shimks** [comment]
@slnode test please

**Comment by shimks** [comment]
@slnode test please

**Pull request merged**